docs/src/pages/demos/menus/MenuListComposition.js

```diff
@@ -60,21 +55,27 @@ class MenuListComposition extends React.Component {
         </Paper>
         <Manager>
           <Target>
-            <Button
-              aria-owns={open ? 'menu-list' : null}
-              aria-haspopup="true"
-              onClick={this.handleClick}
+            <div
+              ref={node => {
+                this.target1 = node;
+              }}
             >
-              Toggle Menu
-            </Button>
+              <Button
+                aria-owns={open ? 'menu-list-grow' : null}
+                aria-haspopup="true"
+                onClick={this.handleToggle}
+              >
+                Toggle Menu Grow
+              </Button>
+            </div>
           </Target>
           <Popper
             placement="bottom-start"
             eventsEnabled={open}
             className={classNames({ [classes.popperClose]: !open })}
           >
             <ClickAwayListener onClickAway={this.handleClose}>
```

Comment:
as a possible optimization, we can do 
```js
const Wrapper = open ? ClickAwayListener : props => props.children;
```
and use `Wrapper` instead of `ClickAwayListener` here, it allows to stop listening when it's closed